app/_includes/head.html

```diff
@@ -158,6 +158,14 @@
     href="/assets/hub.css?v={{ site.time | date: '%s' }}"
   />
   {% endif %}
+
+  {% assign latest_page_url = page.url | replace: page.kong_version, page.kong_latest.release %}
+  {% capture latest_page_exists %}{% page_exists {{latest_page_url}} %}{% endcapture %}
+
+  {% if page.kong_version == site.data.kong_latest.release and page.no_version != true %}<link rel="canonical" href="{{ site.links.web }}{{ page.url }}" />
```

Comment:
```suggestion
  {% if page.kong_version == page.kong_latest.release and page.no_version != true %}<link rel="canonical" href="{{ site.links.web }}{{ page.url }}" />
```
You want this piece of metadata here instead, the one you're using will only pull the latest version for CE gateway - a remnant from the way the site was originally setup with no other expected products/versions.

Reply:
Good to know-thanks